.github/workflows/scripts-versions-2.7.cfg

```diff
@@ -1,2 +1,5 @@
 [versions]
+configparser = 4.0.2
+cryptography = 3.3.2
+keyring = 18.0.1
 more-itertools = 5.0.0
```

Comment:
I'm mildly wondering if it would be a better idea to use `<` pins, e.g. `keyring = <19`, in case a bugfix 18.0.2 ever comes out?

I'm probably overthinking things like I usually do.

Reply:
This is a good idea I was jus following the example provided for `more-itertools`.

Reply:
Done in https://github.com/buildout/buildout/pull/552/commits/538e2d176fcc5f73ebc63e6272103e1b88b6ed8d